templates/page_header/page-header.tpl.php

```diff
@@ -6,8 +6,29 @@
  */
 ?>
 <div class="ecl-page-header">
-  <?php print render($breadcrumb); ?>
+  <?php print render($header_bottom); ?>
```

Comment:
I agree with Imanol, you need to replace '$header_bottom' value with a variable $breadcrumb cf previous comment